rollup-core.config.js

```diff
@@ -59,7 +59,7 @@ export default {
     }),
     checkInternals()
       ? replace({
-          'const __OPIONAL_THEME_DEPS__ = {};': "import { sproutBase } from '@qlik/sprout-theme';",
+          'const __OPIONAL_THEME_DEPS__ = {};': "import { sproutBase } from '@qlik-trial/sprout-theme';",
```

Comment:
Do we still need to do this if we remove the dependency all together?

Reply:
yes, we do. If I understand it correctly when the sn-table is used as a mashup or extension, it will check node_modules/@qlik-trial exists or not first, if it exists, it does use the @qlik-trial/sprout-theme from node_modules.

Reply:
I found that that's one of the reasons, in the sn-table project, we do not directly use `nebula build,` instead, we use `rollup -c rollup.config.js && rollup -c rollup-core.config.js`

Reply:
Yes that is true. But now you are removing the sprout-theme dependency all together. So the check becomes useless since it will never be there right?

Reply:
it also checks node_modules in other projects, right? when it is used as a mashup or extension, it checks node_modules/@qlik-trial in sense-client.

Reply:
I'm not sure what you mean with "used as a mashup"...

These config files are used when running the `yarn build` and `yarn build:watch` commands. Those are run to package the chart. When packaging this you are not running in sense-client in any way so not sure how it checks the node_modules of sense-client. Am I missing something here?